maximum-product-subarray/nhistory.js

```diff
@@ -0,0 +1,16 @@
+var maxProduct = function (nums) {
+  let result = nums[0];
+  let max = 1,
+    min = 1;
+
+  for (let num of nums) {
```

Comment:
이렇게 작성하면, 이 코드를 읽을 때 num이 나중에 재할당 되는 건가~? 하는 생각을 하지 않을 수 있겠네요!

```suggestion
  for (const num of nums) {
```

다른 비슷한 사례는 생략할게요!

Reply:
반영해서 수정하였습니다! 🙇🏻